content/en/tracing/llm_observability/sdk.md

```diff
@@ -307,52 +422,116 @@ def process_message(prompt):
+        ... # user application logic
+        LLMObs.annotate(
+            span=embedding_span,
+            input_data={"text": "Hello world!"},
+            output_data=[0.0023064255, -0.009327292, ...],
+            metrics={"input_tokens": 4},
+            tags={"host": "host_name"},
+        )
+    return
+
+def similarity_search():
+    with LLMObs.retrieval(name="get_relevant_docs") as retrieval_span:
+        ... # user application logic
+        LLMObs.annotate(
+            span=retrieval_span,
+            input_data="Hello world!",
+            output_data=[{"text": "Hello world is ...", "name": "Hello, World! program", "id": "document_id", "score": 0.9893}],
+            tags={"host": "host_name"},
+        )
+    return
+
 {{< /code-block >}}
 
-## Persisting a span across contexts
+## Submitting evaluation metrics
 
-To manually start and stop a span across different contexts or scopes:
+To submit evaluation metrics for a span to Datadog:
 
-1. Start a span manually using the same methods (for example, the `LLMObs.workflow` method for a workflow span), but inline rather than as a context manager. 
-2. Pass the span object as an argument to other functions.
-3. Stop the span manually with the `span.finish()` method.
+1. Extract the span context from the given span by using `LLMObs.export_span(span)`. If `span` is not provided (as when using function decorators), the SDK will export the current active span.
+2. Use `LLMObs.submit_evaluation()` with the extracted span context and evaluation metric information.
 
-### Example
+### Arguments
 
-{{< code-block lang="python" >}}
-from ddtrace.llmobs import LLMObs
+The `LLMObs.submit_evaluation()` method accepts the following arguments:
 
-def separate_task(workflow_span):
-    ... # user application logic 
-    workflow_span.finish()
-    return
+`span_context`
+: required - _dictionary_
+<br />The span context to correlate the evaluation metric to.
+
+`label`
+: required - _string_
+<br />The name of the evaluation metric.
+
+`metric_type`
+: required - _string_
+<br />The type of the evaluation metric. Must be one of "categorical", "numerical", and "score".
+
+`value`
+: required - _string or numeric type_
+<br />The value of the evaluation metric. Must be a string (for categorical `metric_type`) or integer/float (for numerical/score `metric_type`).
+
+## LLM integrations
+
+The Python SDK includes out-of-the-box integrations to automatically trace and annotate the LLM calls for:
+
+- `openai` - OpenAI (using the [OpenAI Python SDK][1]): supports all versions
+- `bedrock` - AWS Bedrock Runtime (using [Boto3][2]/[Botocore][3]): supports all versions
+- `langchain` - LangChain LLM/Chat Models/Chains (using [LangChain][4]): supports all versions
+
+This means that you do not need to manually instrument your LLM calls with `LLMObs.llm()` as the SDK will capture them automatically.
+
+## Advanced tracing
```

Comment:
tiny nitpick. I think function decorators are easier to use, and shouldn't be under "advanced"